src/components/ItemModal/index.js

```diff
@@ -41,6 +44,16 @@ export default class ItemModal extends Component {
           <p className={styles.description}>{displayProperties.description}</p>
         )}
 
+        {$statTrack && (
+          <div>
+            <StatTrack
+              className={styles.objectives}
+              statTrack={$statTrack}
+              description={true}
```

Comment:
So here, consider dropping `description={true}` and instead give it a meaningful `className` to style the text properly.

I think as long as it matches the style and colour of the description that should be good. In fact, maybe even `className={styles.description}` would be good enough 😄